### PR TITLE
[4.4] Cypress typos/grammar

### PR DESCRIPTION
**Pull request by brianteeman**
Cypress typos/grammar

remake of #43833

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 8c92897077e86807319a3e9f6d36f83683717645<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43853">issues.joomla.org/tracker/joomla-cms/43853</a>.</sub>

File: tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ describe('Test in backend that the action logs', () => {
     cy.visit('/administrator/index.php?option=com_actionlogs&view=actionlogs');
   });
 
-  it('have a title', () => {
+  it('has a title', () => {
```
Comment (laoneo) [inline tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js:8]: The whole sentence would be:

_Test in backend that the action logs has a title_

Is this really correct?

Comment (brianteeman) [reply tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js:8]: That is how all the other tests are written

Comment (laoneo) [reply tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js:8]: There is a "have" two tests further down. Basically the "describe" text is the first part of the sentence and the "it" is the end of it. When combined, they should build a meaningful sentence.

Comment (brianteeman) [reply tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js:8]: test in the backend that the page title should contain the text User Action Log

This is referring to a singular test therefore it is has

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 4cd1d3f3b5dff8449b808de5cfb465b67fb4c994<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43853">issues.joomla.org/tracker/joomla-cms/43853</a>.</sub>

**Comment by softforge** [comment]
I have tested this item :white_check_mark: successfully on 4cd1d3f3b5dff8449b808de5cfb465b67fb4c994<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43853">issues.joomla.org/tracker/joomla-cms/43853</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43853">issues.joomla.org/tracker/joomla-cms/43853</a>.</sub>

**Comment by laoneo** [comment]
To the testers here, is this a correct sentence:

_Test in backend that the action logs has a title._

Should it not be:

_Test in backend that the action logs have a title._

**Comment by brianteeman** [comment]
No it should not - We are testing that the page has a title. The page is a singular item therefore it is **has** not **have**

**Comment by laoneo** [comment]
Ok, fine for me then. Probably we should change all occurences then to "Test in backend that the xxx page has yyy." to make it more clear. But this is for another time.

**Pull request merged**